bot.user.js

```diff
@@ -465,14 +461,103 @@ window.collisionCheck = function(circle1, circle2) {
         return false;
     }
 };
-
+// Sort collision points based on distance
+window.getCollisionPoints = function() {
+	var collisionPoints = [];
+    for (var snake in window.snakes){
+		if (window.snakes[snake].nk != window.snake.nk) {
+			for (var pts in window.snakes[snake].pts) {
+				if(!window.snakes[snake].pts[pts].dying){
+					collisionPoint = {xx: window.snakes[snake].pts[pts].xx,
+					 yy: window.snakes[snake].pts[pts].yy,
+					 fx: window.snakes[snake].pts[pts].fx,
+					 fy: window.snakes[snake].pts[pts].fy,
+					 sc: window.snakes[snake].sc,
+					 sp: window.snakes[snake].sp,
+					 };
```

Comment:
This creates a lot of garbage. Perhaps so much that it degrades performance noticably.
Why not just do
```
collisionPoint = window.snakes[snake]
```
instead?